pythran/tables.py

```diff
@@ -4506,6 +4465,92 @@ def partialsum(seq):
     },
 }
 
+if numpy.__version__[0] < '2':
```

Comment:
Instead of duplicating the dict what about the following:

```
def expand_numpy_2_args(*args):
    if numpy.__version__[0] < '2':
          return {'args': args}
    else:
          return {}

# ...
binomial": FunctionIntr(global_effects=true, **expand_numpy_2_args('n', 'p', 'size')),
...
```